docs/devices/index.rst

```diff
@@ -56,6 +56,27 @@ The table below lists what devices and features are supported for each device.
 | Display on device screen           | ✓             | ✓             | ✓          | ✓              | ―        | ✓        | ✓       | ✓        |
 +------------------------------------+---------------+---------------+------------+----------------+----------+----------+---------+----------+
 
+Support Policy
+================
+
+For a device to be supported by HWI, it must:
+
+* Use open source firmware as much as possible
```

Comment:
having trouble installing sphinx on this dumb mac machine, do these nested comments look right to you?

Reply:
Apparently not. Fixed.